specification/_types/aggregations/Aggregate.ts

```diff
@@ -571,7 +573,9 @@ export class SignificantLongTermsBucket extends SignificantTermsBucketBase {
 
 /** @variant name=sigsterms */
 // ES: SignificantStringTerms & InternalSignificantTerms
-export class SignificantStringTermsAggregate extends MultiBucketAggregateBase<SignificantStringTermsBucket> {}
+export class SignificantStringTermsAggregate extends MultiBucketAggregateBase<SignificantStringTermsBucket> {
+  bg_count: long
```

Comment:
I already added that one in https://github.com/elastic/elasticsearch-specification/pull/1940

Since it also appears in other `SignificantXxxAggregate`, we may want to factorize it into a `SignificantTermsAggregateBase<T> extends MultiBucketAggregateBase<T>`.